docs/root/version_history/current.rst

```diff
@@ -1,5 +1,5 @@
-1.20.1 (November 30, 2021)
```

Comment:
This file needs to copied into v1.20.1.rst before modification.

Reply:
Done in 76c6b7939eda59f77e9f9246cc5c1d2009f166ea